ggml/src/ggml-vulkan/ggml-vulkan.cpp

```diff
@@ -6029,6 +6174,17 @@ static void ggml_vk_mul_mat_vec_q_f16(ggml_backend_vk_context * ctx, vk_context&
             ctx->prealloc_y_last_tensor_used = src1;
         }
     }
+    if (quantize_y) {
+        if (ctx->prealloc_y_last_pipeline_used != to_q8_1.get() ||
+            ctx->prealloc_y_last_tensor_used != src1) {
+            if (ctx->prealloc_y_need_sync) {
```

Comment:
At line 6231, add `|| quantize_y` (github won't let me make a suggestion there).

Reply:
Oh yeah, I hadn't seen that.